packages/nocodb/src/db/field-handler/index.ts

```
@@ -0,0 +1,340 @@
+
+  async verifyFilters(filters: Filter[], options: HandlerOptions = {}) {
+    const verificationResult = await this.verifyFiltersSafe(filters, options);
+    if (!verificationResult.isValid) {
+      throw new FilterVerificationError(verificationResult.errors!);
+    }
+    return true;
+  }
+
+  async traverseFilters(
+    filters: Filter[],
+    handler: <T>(filter: Filter) => Promise<void | T>,
```

Biome (1.9.4)

[error] void is confusing inside a union type.

Unsafe fix: Use undefined instead.


(lint/suspicious/noConfusingVoidType)